README.md

```diff
@@ -1 +1,43 @@
-# iqra-web
\ No newline at end of file
+# Iqra website
+
+Iqra is a tool meant to allow Muslims to search the Quran using speech recognition. This repo contains the code for the [website](https://iqraapp.com). There are also repos for the [Android client](https://github.com/Crescent-Labs/iqra-android) and [iOS client](https://github.com/Crescent-Labs/iqra-ios).
+
+### Setup
+
+The website's back-end is a [Flask](http://flask.pocoo.org/) server, and the front-end is [React](https://facebook.github.io/react/) combined with [Redux](http://redux.js.org/). To run the project locally, first clone/download this repo and then navigate into the root of the directory. Then, you'll need to install the Python and JavaScript dependencies:
+
+```
+pip install -r requirements.txt
+npm install
+```
+
+The next step is to duplicate the `default_config.py` file and name the copy `config.py`. In `config.py`, you'll need to replace the `APIKEY` variable's value with a valid key. You can obtain a key for testing by sending an email to info@iqraapp.com with subject line "api key request". Please allow some time for your request to be processed and an api key sent back.
```

Comment:
No need. It's automatically linked. When reviewing the changed files click "view" in the top right of the file.